site/content/docs/5.0/migration.md

```diff
@@ -7,6 +7,15 @@ aliases: "/migration/"
 toc: true
 ---
 
+## v5.0.0-alpha3
+
+### Colors
+
+- The color system which worked with `color-level()` and `$theme-color-interval` was removed in favor of a new color system.
+- All `lighten()` and `darken()` functions in our codebase are replaced by `tint-color()` and `shade-color()`. These functions provide will mix a color with either white or black instead of changing the lightness by a fixed amount.
```

Comment:
"These functions provide will mix" seems weird, I guess "provide" should be dropped?